docs/index.adoc

```diff
@@ -13185,6 +13184,12 @@ ____
 
 ====
 
+=== Spring Boot Maven Plugin
+
+The https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging[Spring Boot Maven Plugin] provides a `repakcage` goal to create executable archives that contain all of an application’s dependencies. It thus functions as an executable, is portable across operating systems and architectures, and is still a valid JAR file that can be executed explicitly or otherwise used as a JAR.
```

Comment:
> provides a `repakcage` goal

typo? `repackage`?

Reply:
```suggestion
The https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging[Spring Boot Maven Plugin] provides a `repackage` goal to create executable archives that contain all of an application’s dependencies. It thus functions as an executable, is portable across operating systems and architectures, and is still a valid JAR file that can be executed explicitly or otherwise used as a JAR.
```